apps/src/music/TopButtons.jsx

```diff
@@ -64,12 +73,30 @@ const TopButtons = ({clearCode}) => {
         <FontAwesome icon={'commenting'} />
         &nbsp; Feedback
       </button>
+      <fieldset>
```

Comment:
Similarly here - maybe a separate `UploadButton` React component? Might be a bit overkill though, i'll leave it up to you.